test/e2e/tests/settings-search.spec.js

```diff
@@ -22,6 +22,8 @@ describe('Settings Search', function () {
     about: 'Terms of Use',
   };
 
+  let page;
```

Comment:
[Q] Non blocking, but why use reassignment on this variable versus just declaring it as a constant in each test case?

Reply:
that's a good point! I've updated it with `const` variables inside each testcase scope @brad-decker  ty